### ISSUE TITLE
Color contrast: UI components

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-modules/CSSEditor#12]
UI components must have a contrast of at least 3:1 to their surroundings, though must components are not currently meeting this minimum standard: 
- `.remove-value` buttons are okay on hover (at 3.7:1), but in their normal state their contrast is only 2.6:1.
- The `.add-value` button uses an opacity in its background that currently puts the text to background relationship below where we want it to be (4.4:1) as it is situated over a div with its own background color. "Save" submit buttons use the same styling, but they are on a lighter background and thus are at an okay contrast level, though that may need to change as well for the sake of consistency.
- All input areas (for CSS and external stylesheets) have borders that fail contrast needs for adjacent content. They are 1.4:1 with their #FFFFFF text/textarea backgrounds and 1.3:1 with their surroundings.
